HISTORY.rst

```diff
@@ -41,6 +41,10 @@ Unreleased Changes (3.9)
     * ``convolve_2d`` keyword ``ignore_nodata`` to ``ignore_nodata_and_edges``.
     * ``get_raster_info`` / ``get_vector_info`` keyword ``projection`` to
       ``projection_wkt``.
+* SDR:
+  * Fixing an issue where the LS factor should be capped to 333 m^2 upstream.
```

Comment:
Here's another place to just confirm whether it should be 333 square meters or 333**2 square meters.

Reply:
Fixed in fc2cebe6bf3357e68b8f5383e3adbaa9c3fbef74